### ISSUE TITLE
Watch regression in Horizon 1.3

### ISSUE THREAD

**Issue by yankeeinlondon** [rethinkdb/horizon#610]
I have been able to use the `watch()` functionality in Horizon with prior releases but having upgraded yesterday to **1.3** I now am unable to leverage **any** of the "real time" features of Horizon's "watches". 

Specifically I have a very simple two table model: "person" and "todo". If I use the web console in either Safari or Chrome and type:

``` js
hz = window.Horizon();
hz('person').watch().subscribe(changes => console.log(changes))
```

I would expect to get back the list of changes as they happen in real time. However, while I get back a returned collection object:

![image](https://cloud.githubusercontent.com/assets/980757/16222138/7b98c56e-378e-11e6-922e-8b88fbc15457.png)

followed by two errors:

![2016-06-21_09-04-40](https://cloud.githubusercontent.com/assets/980757/16222328/63a2b540-378f-11e6-9f57-9b48704a572d.png)

and immediately following that error is ...

![2016-06-21_09-08-45](https://cloud.githubusercontent.com/assets/980757/16222433/e176b462-378f-11e6-9346-0423921db6f4.png)

While I understand there is movement-afoot to move away from `engine.io` (it was suggested that this issue might be related to that) ... I would point out the prior releases of Horizon worked just fine with this functionality so I suspect it isn't entirely an `engine.io` issue but rather some sort of introduced regression.

> **Note:** not only did I upgrade to Horizon 1.3 yesterday but I also upgraded the DB to latest ReThink DB

**Comment by deontologician** [comment]
Hey @ksnyde I'll check this out today

**Comment by yankeeinlondon** [comment]
@deontologician thanks for giving it the attention but I hope you haven't spent time on it yet as this is a red herring. The browser error was a result of my lack of configuration when I initialized Horizon (specifically the lack of `host` property).

Once I realized the code problem was separate from the web debugger issue I have made some good progress the past hour or two. What remain issues for me are now EmberJS issues not Horizon ones. :)
